File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Java/src/main/java/simulacionCesta/simulacionCestaCompra.java
package simulacionCesta;

import simulacionCesta.factories.FinalCestFactory;
import simulacionCesta.models.FinalCest;

import static simulacionCesta.utils.funcionesMenuCesta.menuCest;

public class simulacionCestaCompra {
    public static void main(String[] args) throws InterruptedException {
        // Inicializamos la línea de cesta que contiene objetos aleatorios
        FinalCest cest = FinalCestFactory.create();
        // Iniciamos menú
        menuCest(cest);
    }
}

File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Kotlin/src/main/kotlin/simulacionCesta/simulacionCestaCompra.kt
package simulacionCesta

import simulacionCesta.factories.FinalCestFactory
import simulacionCesta.utils.menuCest

fun main(args: Array<String>) {
    // Inicializamos la línea de cesta que contiene objetos aleatorios
    val cest = FinalCestFactory.create()
    // Iniciamos menú
    menuCest(cest)
}